models/error.go

```diff
@@ -4,7 +4,9 @@
 
 package models
 
-import "fmt"
+import (
```

Comment:
In addition to the gogs copyright at the head of this file, could you add the one for gitea too?

Reply:
Yeah, will do. Feel free to review now, even though I have it in draft. Just I know I need to myself go over everything with a fine comb as comments and such could be improved.